web/packages/teleterm/notarize.js

```diff
@@ -1,36 +0,0 @@
-const { notarize } = require('@electron/notarize');
-
-exports.default = async function notarizing(context) {
-  const { electronPlatformName, appOutDir } = context;
-  if (electronPlatformName !== 'darwin') {
-    return;
-  }
-
-  if (!process.env.APPLE_USERNAME || !process.env.APPLE_PASSWORD) {
-    console.warn(
-      'missing $APPLE_USERNAME and/or $APPLE_PASSWORD: notarization will be skipped'
-    );
-    return;
-  }
-
-  if (!process.env.TEAMID) {
-    console.warn(
-      'missing $TEAMID: notarization will be skipped. Run `make release-connect` instead'
```

Comment:
What's the difference in output now when notarization is done and when it's skipped?

Reply:
When it's done:
```
  • signing         file=build/release/mac-universal/Teleport Connect.app platform=darwin type=distribution provisioningProfile=none
  • notarization successful
  • building        target=DMG arch=universal file=build/release/Teleport Connect-19.0.0-dev.gzdunek.3-universal.dmg
```

When it's skipped:
```
  • signing         file=build/release/mac-arm64/Teleport Connect.app platform=darwin type=distribution provisioningProfile=none
  • skipped macOS notarization  reason=`notarize` options were unable to be generated
  • building        target=DMG arch=arm64 file=build/release/Teleport Connect-1.0.0-dev-arm64.dmg
```